### PR TITLE
Feedback Now Emails Innovate

### PR DESCRIPTION
**Pull request by davethenipper**
Removed the link in feedback to "Innovate" idea on innovate.  Feedback
button now emails innovate@ra.rockwell.com

Addresses issue #344

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -110,7 +110,7 @@
 
             <div class="md-padding" layout="row">
                 <!-- Feedback Button -->
-                <md-button class="md-icon-button" aria-label="Feedback" ui-sref="idea({ ideaId: '566eff2a6405f74421c3a583' })" ng-if="isUserLoggedIn()">
+                <md-button class="md-icon-button" aria-label="Feedback" ng-if="isUserLoggedIn()" href="mailto:innovate@ra.rockwell.com">
```
Comment (alanlgirard) [inline src/index.html]: Should probably include a subject line such as "Innovate Feedback" in the href so we have a default subject to filter on, etc.

**Comment by davethenipper** [comment]
That is a really good idea.  I'll add that in after I pull in your changes to fix the build.

**Comment by davethenipper** [comment]
Ready for approvals @alanlgirard @YashdalfTheGray @monotkate

**Comment by alanlgirard** [comment]
Approved

**Comment by davethenipper** [comment]
I approve

**Comment by davethenipper** [comment]
That's 2!

Merging

**Comment by davethenipper** [comment]
Salt N peppa this

**Comment by davethenipper** [comment]
#PushIt

**Pull request merged**